render/picturestr.h

```diff
@@ -39,8 +39,8 @@ typedef struct _DirectFormat {
 
 typedef struct _IndexFormat {
     VisualID vid;
-    ColormapPtr pColormap;
     int nvalues;
+    ColormapPtr pColormap;
```

Comment:
part of public ABI and likely used by proprietary Nvidia driver, so we can't change that.